source/fundamentals/connection.txt

```diff
@@ -496,6 +497,12 @@ parameters of the connection URI to specify the behavior of the client.
      - boolean
      - Specifies that the driver must connect to the host directly.
 
+   * - **maxConnecting**
+     - integer
+     - Specifies the number of connections the driver concurrently
```

Comment:
It's closer to this: "Specifies the maximum number of connections to each server that the driver will concurrently establish before making them available in the connection pool."